dataflux_pytorch/dataflux_iterable_dataset.py

```diff
@@ -111,53 +114,66 @@ def __init__(
                 during initialization.
         """
         super().__init__()
-        self.storage_client = storage_client
-        if not storage_client:
-            self.storage_client = storage.Client(project=project_name, )
-        dataflux_core.user_agent.add_dataflux_user_agent(self.storage_client)
+        self.storage_client = None
+        if OS == LINUX:
+            self.storage_client = (
+                storage_client
+                if storage_client is not None
+                else storage.Client(project=project_name)
+            )
+            dataflux_core.user_agent.add_dataflux_user_agent(self.storage_client)
         self.project_name = project_name
         self.bucket_name = bucket_name
-        self.data_format_fn = data_format_fn
+        self.data_format_fn = (
+            data_format_fn if data_format_fn is not None else self.data_format_default
+        )
         self.config = config
         self.dataflux_download_optimization_params = (
             dataflux_core.download.DataFluxDownloadOptimizationParams(
                 max_composite_object_size=self.config.max_composite_object_size
-            ))
+            )
+        )
 
         self.objects = self._list_GCS_blobs_with_retry()
 
     def __iter__(self):
         worker_info = data.get_worker_info()
         if worker_info is None:
             # Single-process data loading.
-            yield from (self.data_format_fn(bytes_content) for bytes_content in
-                        dataflux_core.download.dataflux_download_lazy(
-                            project_name=self.project_name,
-                            bucket_name=self.bucket_name,
-                            objects=self.objects,
-                            storage_client=self.storage_client,
-                            dataflux_download_optimization_params=self.
-                            dataflux_download_optimization_params,
-                            retry_config=self.config.download_retry_config,
-                        ))
+            yield from (
```

Comment:
Here and elsewhere in this PR, it looks like your formatter might have reformatted some things with different settings than we're using for the project. Can you make sure that the recommended extensions are installed (Extensions > Recommended in VS Code) and then re-run the formatter on these files?